### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by ifarooq09**
assignment for week 3 completed as per instructions and submitting for review

### CODE REVIEW

**Comment by rohwedernt** [comment]
It looks like all the files changed in the course are included here. As those files are merged into main and as you work of the current main locally the changed files here should only reflect the new work for lesson 3. Git can be tricky! Let me know if this doesn't make sense and Ill be happy to help.

**Comment by ifarooq09** [comment]
Hi Nate, Sorry I didn't get you. Could you please explain it?

**Comment by rohwedernt** [comment]
This pull request represents the files changed compared to the main branch that you are "requesting" to push the changed code into. For each lesson you open a pull request and merge it into the main branch. In this way, the pull request should only show changes for the files for that lesson (since everything else is in main). For this lesson that is something like index, app, settings and the package-json. Instead it looks like this includes the changes for all the lessons up to this point. It looks, oddly, like those answers haven't made it into main yet for your fork so lets not worry about this for now. Ill review the week 3 material here and once I approve it, make sure you click the merge button here in github. After that ensure you "pull" this updated main down to your local main branch so that when you add your week 4 answers you are working against the current code. [This site](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners) has a good overview of git strategy if you want to do any further reading.

**Review by rohwedernt — APPROVED** [review]

**Pull request merged**
